packages/cursorless-org-docs/src/docs/contributing/CONTRIBUTING.mdx

```diff
@@ -10,7 +12,8 @@ extension](#running--testing-extension-locally). You may also find the [VSCode A
 - [Git](https://git-scm.com/)
 - [Node.js](https://nodejs.org/en/)
 - [Corepack](https://nodejs.org/api/corepack.html) or [pnpm](https://pnpm.io/installation)
-- [VSCode](https://code.visualstudio.com/); minimum version for local development is 1.72.0 in order to support settings profiles for sandboxed development. Please file an issue if that is a problem.
+- [VSCode](https://code.visualstudio.com/) or [VSCodium](https://vscodium.com/); Please file an issue if you are unable to use [settings profiles](https://code.visualstudio.com/updates/v1_72#_settings-profiles), as they are required for the development environment.
```

Comment:
Not sure what the point is about filing an issue anymore. I would just remove that part.